.github/workflows/android.yml

```diff
@@ -21,4 +21,4 @@ jobs:
         java-version: '11'
         distribution: 'adopt'
     - name: Build with Gradle
-      run: ./gradlew build jacocoTestReport -x lint --stacktrace
```

Comment:
hmm actually shouldn't we keep this and just remove `-x`?

Reply:
`lint` is part of the task graph for `build`, so we don't need to specify it explicitly here. `build` includes `assemble` and `check`, and `lint` falls under `check`. The `-x lint` is just general Gradle convention for explicitly preventing a task to be run.

These dependencies are buried in the http://tools.android.com/tech-docs/new-build-system/user-guide#TOC-Build-Tasks docs, but looking at this [gradle task info plugin](https://gitlab.com/barfuin/gradle-taskinfo#gradle-taskinfo) the output showing which tasks are run for `build` confirms it:

```
$ ./gradlew tiOrder build

...

> Task :tiOrder

In order to execute [:demo:build, :library-v3:build, :library:build, :lint-checks:build], the following tasks would be executed in this order:

  1. :demo:preBuild                                    (org.gradle.api.DefaultTask)
  2. :demo:preGmsDebugBuild                            (com.android.build.gradle.internal.tasks.AndroidVariantTask)
  3. :demo:compileGmsDebugAidl                         (com.android.build.gradle.tasks.AidlCompile)
  4. :demo:compileGmsDebugRenderscript                 (com.android.build.gradle.tasks.RenderscriptCompile)
  5. :demo:generateGmsDebugBuildConfig                 (com.android.build.gradle.tasks.GenerateBuildConfig)
  6. :demo:checkGmsDebugAarMetadata                    (com.android.build.gradle.internal.tasks.CheckAarMetadataTask)
  7. :demo:generateGmsDebugResValues                   (com.android.build.gradle.tasks.GenerateResValues)
  8. :demo:generateGmsDebugResources                   (org.gradle.api.DefaultTask)
  9. :demo:mergeGmsDebugResources                      (com.android.build.gradle.tasks.MergeResources)
 10. :demo:createGmsDebugCompatibleScreenManifests     (com.android.build.gradle.tasks.CompatibleScreensManifest)
 11. :demo:extractDeepLinksGmsDebug                    (com.android.build.gradle.tasks.ExtractDeepLinksTask)
 12. :demo:processGmsDebugMainManifest                 (com.android.build.gradle.tasks.ProcessApplicationManifest)
 13. :demo:processGmsDebugManifest                     (com.android.build.gradle.tasks.ProcessMultiApkApplicationManifest)
 14. :demo:processGmsDebugManifestForPackage           (com.android.build.gradle.tasks.ProcessPackagedManifestTask)
 15. :demo:processGmsDebugResources                    (com.android.build.gradle.internal.res.LinkApplicationAndroidResourcesTask)
 16. :demo:compileGmsDebugKotlin                       (org.jetbrains.kotlin.gradle.tasks.KotlinCompileWithWorkers)
 17. :demo:javaPreCompileGmsDebug                      (com.android.build.gradle.tasks.JavaPreCompileTask)
 18. :demo:compileGmsDebugJavaWithJavac                (org.gradle.api.tasks.compile.JavaCompile)
 19. :demo:compileGmsDebugSources                      (org.gradle.api.Task)
 20. :demo:mergeGmsDebugNativeDebugMetadata            (com.android.build.gradle.internal.tasks.MergeNativeDebugMetadataTask)
 21. :demo:mergeGmsDebugShaders                        (com.android.build.gradle.tasks.MergeSourceSetFolders)
 22. :demo:compileGmsDebugShaders                      (com.android.build.gradle.tasks.ShaderCompile)
 23. :demo:generateGmsDebugAssets                      (org.gradle.api.DefaultTask)
 24. :demo:mergeGmsDebugAssets                         (com.android.build.gradle.tasks.MergeSourceSetFolders)
 25. :demo:compressGmsDebugAssets                      (com.android.build.gradle.internal.tasks.CompressAssetsTask)
 26. :demo:checkGmsDebugDuplicateClasses               (com.android.build.gradle.internal.tasks.CheckDuplicateClassesTask)
 27. :demo:dexBuilderGmsDebug                          (com.android.build.gradle.internal.tasks.DexArchiveBuilderTask)
 28. :demo:desugarGmsDebugFileDependencies             (com.android.build.gradle.internal.tasks.DexFileDependenciesTask)
 29. :demo:mergeExtDexGmsDebug                         (com.android.build.gradle.internal.tasks.DexMergingTask)
 30. :demo:mergeDexGmsDebug                            (com.android.build.gradle.internal.tasks.DexMergingTask)
 31. :demo:processGmsDebugJavaRes                      (com.android.build.gradle.internal.tasks.ProcessJavaResTask)
 32. :demo:mergeGmsDebugJavaResource                   (com.android.build.gradle.internal.tasks.MergeJavaResourceTask)
 33. :demo:mergeGmsDebugJniLibFolders                  (com.android.build.gradle.tasks.MergeSourceSetFolders)
 34. :demo:mergeGmsDebugNativeLibs                     (com.android.build.gradle.internal.tasks.MergeNativeLibsTask)
 35. :demo:stripGmsDebugDebugSymbols                   (com.android.build.gradle.internal.tasks.StripDebugSymbolsTask)
 36. :demo:validateSigningGmsDebug                     (com.android.build.gradle.internal.tasks.ValidateSigningTask)
 37. :demo:writeGmsDebugAppMetadata                    (com.android.build.gradle.internal.tasks.AppMetadataTask)
 38. :demo:writeGmsDebugSigningConfigVersions          (com.android.build.gradle.internal.tasks.SigningConfigVersionsWriterTask)
 39. :demo:packageGmsDebug                             (com.android.build.gradle.tasks.PackageApplication)
 40. :demo:assembleGmsDebug                            (org.gradle.api.Task)
 41. :demo:preV3DebugBuild                             (com.android.build.gradle.internal.tasks.AndroidVariantTask)
 42. :demo:compileV3DebugAidl                          (com.android.build.gradle.tasks.AidlCompile)
 43. :demo:compileV3DebugRenderscript                  (com.android.build.gradle.tasks.RenderscriptCompile)
 44. :demo:generateV3DebugBuildConfig                  (com.android.build.gradle.tasks.GenerateBuildConfig)
 45. :demo:checkV3DebugAarMetadata                     (com.android.build.gradle.internal.tasks.CheckAarMetadataTask)
 46. :demo:generateV3DebugResValues                    (com.android.build.gradle.tasks.GenerateResValues)
 47. :demo:generateV3DebugResources                    (org.gradle.api.DefaultTask)
 48. :demo:mergeV3DebugResources                       (com.android.build.gradle.tasks.MergeResources)
 49. :demo:createV3DebugCompatibleScreenManifests      (com.android.build.gradle.tasks.CompatibleScreensManifest)
 50. :demo:extractDeepLinksV3Debug                     (com.android.build.gradle.tasks.ExtractDeepLinksTask)
 51. :demo:processV3DebugMainManifest                  (com.android.build.gradle.tasks.ProcessApplicationManifest)
 52. :demo:processV3DebugManifest                      (com.android.build.gradle.tasks.ProcessMultiApkApplicationManifest)
 53. :demo:processV3DebugManifestForPackage            (com.android.build.gradle.tasks.ProcessPackagedManifestTask)
 54. :demo:processV3DebugResources                     (com.android.build.gradle.internal.res.LinkApplicationAndroidResourcesTask)
 55. :demo:compileV3DebugKotlin                        (org.jetbrains.kotlin.gradle.tasks.KotlinCompileWithWorkers)
 56. :demo:javaPreCompileV3Debug                       (com.android.build.gradle.tasks.JavaPreCompileTask)
 57. :demo:compileV3DebugJavaWithJavac                 (org.gradle.api.tasks.compile.JavaCompile)
 58. :demo:compileV3DebugSources                       (org.gradle.api.Task)
 59. :demo:mergeV3DebugNativeDebugMetadata             (com.android.build.gradle.internal.tasks.MergeNativeDebugMetadataTask)
 60. :demo:mergeV3DebugShaders                         (com.android.build.gradle.tasks.MergeSourceSetFolders)
 61. :demo:compileV3DebugShaders                       (com.android.build.gradle.tasks.ShaderCompile)
 62. :demo:generateV3DebugAssets                       (org.gradle.api.DefaultTask)
 63. :demo:mergeV3DebugAssets                          (com.android.build.gradle.tasks.MergeSourceSetFolders)
 64. :demo:compressV3DebugAssets                       (com.android.build.gradle.internal.tasks.CompressAssetsTask)
 65. :demo:checkV3DebugDuplicateClasses                (com.android.build.gradle.internal.tasks.CheckDuplicateClassesTask)
 66. :demo:desugarV3DebugFileDependencies              (com.android.build.gradle.internal.tasks.DexFileDependenciesTask)
 67. :demo:dexBuilderV3Debug                           (com.android.build.gradle.internal.tasks.DexArchiveBuilderTask)
 68. :demo:mergeDexV3Debug                             (com.android.build.gradle.internal.tasks.DexMergingTask)
 69. :demo:processV3DebugJavaRes                       (com.android.build.gradle.internal.tasks.ProcessJavaResTask)
 70. :demo:mergeV3DebugJavaResource                    (com.android.build.gradle.internal.tasks.MergeJavaResourceTask)
 71. :demo:mergeV3DebugJniLibFolders                   (com.android.build.gradle.tasks.MergeSourceSetFolders)
 72. :demo:mergeV3DebugNativeLibs                      (com.android.build.gradle.internal.tasks.MergeNativeLibsTask)
 73. :demo:stripV3DebugDebugSymbols                    (com.android.build.gradle.internal.tasks.StripDebugSymbolsTask)
 74. :demo:validateSigningV3Debug                      (com.android.build.gradle.internal.tasks.ValidateSigningTask)
 75. :demo:writeV3DebugAppMetadata                     (com.android.build.gradle.internal.tasks.AppMetadataTask)
 76. :demo:writeV3DebugSigningConfigVersions           (com.android.build.gradle.internal.tasks.SigningConfigVersionsWriterTask)
 77. :demo:packageV3Debug                              (com.android.build.gradle.tasks.PackageApplication)
 78. :demo:assembleV3Debug                             (org.gradle.api.Task)
 79. :demo:assembleDebug                               (org.gradle.api.Task)
 80. :demo:preGmsReleaseBuild                          (com.android.build.gradle.internal.tasks.AndroidVariantTask)
 81. :demo:compileGmsReleaseAidl                       (com.android.build.gradle.tasks.AidlCompile)
 82. :demo:compileGmsReleaseRenderscript               (com.android.build.gradle.tasks.RenderscriptCompile)
 83. :demo:generateGmsReleaseBuildConfig               (com.android.build.gradle.tasks.GenerateBuildConfig)
 84. :demo:checkGmsReleaseAarMetadata                  (com.android.build.gradle.internal.tasks.CheckAarMetadataTask)
 85. :demo:generateGmsReleaseResValues                 (com.android.build.gradle.tasks.GenerateResValues)
 86. :demo:generateGmsReleaseResources                 (org.gradle.api.DefaultTask)
 87. :demo:mergeGmsReleaseResources                    (com.android.build.gradle.tasks.MergeResources)
 88. :demo:createGmsReleaseCompatibleScreenManifests   (com.android.build.gradle.tasks.CompatibleScreensManifest)
 89. :demo:extractDeepLinksGmsRelease                  (com.android.build.gradle.tasks.ExtractDeepLinksTask)
 90. :demo:processGmsReleaseMainManifest               (com.android.build.gradle.tasks.ProcessApplicationManifest)
 91. :demo:processGmsReleaseManifest                   (com.android.build.gradle.tasks.ProcessMultiApkApplicationManifest)
 92. :demo:processGmsReleaseManifestForPackage         (com.android.build.gradle.tasks.ProcessPackagedManifestTask)
 93. :demo:processGmsReleaseResources                  (com.android.build.gradle.internal.res.LinkApplicationAndroidResourcesTask)
 94. :demo:compileGmsReleaseKotlin                     (org.jetbrains.kotlin.gradle.tasks.KotlinCompileWithWorkers)
 95. :demo:javaPreCompileGmsRelease                    (com.android.build.gradle.tasks.JavaPreCompileTask)
 96. :demo:writeGmsReleaseApplicationId                (com.android.build.gradle.internal.tasks.ApplicationIdWriterTask)
 97. :demo:analyticsRecordingGmsRelease                (com.android.build.gradle.internal.tasks.AnalyticsRecordingTask)
 98. :demo:compileGmsReleaseJavaWithJavac              (org.gradle.api.tasks.compile.JavaCompile)
 99. :demo:compileGmsReleaseSources                    (org.gradle.api.Task)
100. :demo:extractProguardFiles                        (com.android.build.gradle.internal.tasks.ExtractProguardFiles)
101. :demo:bundleGmsReleaseClasses                     (com.android.build.gradle.internal.feature.BundleAllClasses)
102. :demo:lintVitalAnalyzeGmsRelease                  (com.android.build.gradle.internal.lint.AndroidLintAnalysisTask)
103. :demo:lintVitalGmsRelease                         (com.android.build.gradle.internal.lint.AndroidLintTask)
104. :demo:mergeGmsReleaseJniLibFolders                (com.android.build.gradle.tasks.MergeSourceSetFolders)
105. :demo:mergeGmsReleaseNativeLibs                   (com.android.build.gradle.internal.tasks.MergeNativeLibsTask)
106. :demo:stripGmsReleaseDebugSymbols                 (com.android.build.gradle.internal.tasks.StripDebugSymbolsTask)
107. :demo:extractGmsReleaseNativeSymbolTables         (com.android.build.gradle.internal.tasks.ExtractNativeDebugMetadataTask)
108. :demo:mergeGmsReleaseNativeDebugMetadata          (com.android.build.gradle.internal.tasks.MergeNativeDebugMetadataTask)
109. :demo:mergeGmsReleaseArtProfile                   (com.android.build.gradle.internal.tasks.MergeArtProfileTask)
110. :demo:checkGmsReleaseDuplicateClasses             (com.android.build.gradle.internal.tasks.CheckDuplicateClassesTask)
111. :demo:mergeGmsReleaseGeneratedProguardFiles       (com.android.build.gradle.internal.tasks.MergeFileTask)
112. :demo:processGmsReleaseJavaRes                    (com.android.build.gradle.internal.tasks.ProcessJavaResTask)
113. :demo:mergeGmsReleaseJavaResource                 (com.android.build.gradle.internal.tasks.MergeJavaResourceTask)
114. :demo:minifyGmsReleaseWithR8                      (com.android.build.gradle.internal.tasks.R8Task)
115. :demo:compileGmsReleaseArtProfile                 (com.android.build.gradle.internal.tasks.CompileArtProfileTask)
116. :demo:mergeGmsReleaseShaders                      (com.android.build.gradle.tasks.MergeSourceSetFolders)
117. :demo:compileGmsReleaseShaders                    (com.android.build.gradle.tasks.ShaderCompile)
118. :demo:generateGmsReleaseAssets                    (org.gradle.api.DefaultTask)
119. :demo:mergeGmsReleaseAssets                       (com.android.build.gradle.tasks.MergeSourceSetFolders)
120. :demo:compressGmsReleaseAssets                    (com.android.build.gradle.internal.tasks.CompressAssetsTask)
121. :demo:optimizeGmsReleaseResources                 (com.android.build.gradle.internal.tasks.OptimizeResourcesTask)
122. :demo:collectGmsReleaseDependencies               (com.android.build.gradle.internal.tasks.PerModuleReportDependenciesTask)
123. :demo:sdkGmsReleaseDependencyData                 (com.android.build.gradle.internal.tasks.SdkDependencyDataGeneratorTask)
124. :demo:writeGmsReleaseAppMetadata                  (com.android.build.gradle.internal.tasks.AppMetadataTask)
125. :demo:writeGmsReleaseSigningConfigVersions        (com.android.build.gradle.internal.tasks.SigningConfigVersionsWriterTask)
126. :demo:packageGmsRelease                           (com.android.build.gradle.tasks.PackageApplication)
127. :demo:assembleGmsRelease                          (org.gradle.api.Task)
128. :demo:assembleGms                                 (org.gradle.api.Task)
129. :demo:preV3ReleaseBuild                           (com.android.build.gradle.internal.tasks.AndroidVariantTask)
130. :demo:compileV3ReleaseAidl                        (com.android.build.gradle.tasks.AidlCompile)
131. :demo:compileV3ReleaseRenderscript                (com.android.build.gradle.tasks.RenderscriptCompile)
132. :demo:generateV3ReleaseBuildConfig                (com.android.build.gradle.tasks.GenerateBuildConfig)
133. :demo:checkV3ReleaseAarMetadata                   (com.android.build.gradle.internal.tasks.CheckAarMetadataTask)
134. :demo:generateV3ReleaseResValues                  (com.android.build.gradle.tasks.GenerateResValues)
135. :demo:generateV3ReleaseResources                  (org.gradle.api.DefaultTask)
136. :demo:mergeV3ReleaseResources                     (com.android.build.gradle.tasks.MergeResources)
137. :demo:createV3ReleaseCompatibleScreenManifests    (com.android.build.gradle.tasks.CompatibleScreensManifest)
138. :demo:extractDeepLinksV3Release                   (com.android.build.gradle.tasks.ExtractDeepLinksTask)
139. :demo:processV3ReleaseMainManifest                (com.android.build.gradle.tasks.ProcessApplicationManifest)
140. :demo:processV3ReleaseManifest                    (com.android.build.gradle.tasks.ProcessMultiApkApplicationManifest)
141. :demo:processV3ReleaseManifestForPackage          (com.android.build.gradle.tasks.ProcessPackagedManifestTask)
142. :demo:processV3ReleaseResources                   (com.android.build.gradle.internal.res.LinkApplicationAndroidResourcesTask)
143. :demo:compileV3ReleaseKotlin                      (org.jetbrains.kotlin.gradle.tasks.KotlinCompileWithWorkers)
144. :demo:javaPreCompileV3Release                     (com.android.build.gradle.tasks.JavaPreCompileTask)
145. :demo:writeV3ReleaseApplicationId                 (com.android.build.gradle.internal.tasks.ApplicationIdWriterTask)
146. :demo:analyticsRecordingV3Release                 (com.android.build.gradle.internal.tasks.AnalyticsRecordingTask)
147. :demo:compileV3ReleaseJavaWithJavac               (org.gradle.api.tasks.compile.JavaCompile)
148. :demo:compileV3ReleaseSources                     (org.gradle.api.Task)
149. :demo:bundleV3ReleaseClasses                      (com.android.build.gradle.internal.feature.BundleAllClasses)
150. :demo:lintVitalAnalyzeV3Release                   (com.android.build.gradle.internal.lint.AndroidLintAnalysisTask)
151. :demo:lintVitalV3Release                          (com.android.build.gradle.internal.lint.AndroidLintTask)
152. :demo:mergeV3ReleaseJniLibFolders                 (com.android.build.gradle.tasks.MergeSourceSetFolders)
153. :demo:mergeV3ReleaseNativeLibs                    (com.android.build.gradle.internal.tasks.MergeNativeLibsTask)
154. :demo:stripV3ReleaseDebugSymbols                  (com.android.build.gradle.internal.tasks.StripDebugSymbolsTask)
155. :demo:extractV3ReleaseNativeSymbolTables          (com.android.build.gradle.internal.tasks.ExtractNativeDebugMetadataTask)
156. :demo:mergeV3ReleaseNativeDebugMetadata           (com.android.build.gradle.internal.tasks.MergeNativeDebugMetadataTask)
157. :demo:mergeV3ReleaseArtProfile                    (com.android.build.gradle.internal.tasks.MergeArtProfileTask)
158. :demo:checkV3ReleaseDuplicateClasses              (com.android.build.gradle.internal.tasks.CheckDuplicateClassesTask)
159. :demo:mergeV3ReleaseGeneratedProguardFiles        (com.android.build.gradle.internal.tasks.MergeFileTask)
160. :demo:processV3ReleaseJavaRes                     (com.android.build.gradle.internal.tasks.ProcessJavaResTask)
161. :demo:mergeV3ReleaseJavaResource                  (com.android.build.gradle.internal.tasks.MergeJavaResourceTask)
162. :demo:minifyV3ReleaseWithR8                       (com.android.build.gradle.internal.tasks.R8Task)
163. :demo:compileV3ReleaseArtProfile                  (com.android.build.gradle.internal.tasks.CompileArtProfileTask)
164. :demo:mergeV3ReleaseShaders                       (com.android.build.gradle.tasks.MergeSourceSetFolders)
165. :demo:compileV3ReleaseShaders                     (com.android.build.gradle.tasks.ShaderCompile)
166. :demo:generateV3ReleaseAssets                     (org.gradle.api.DefaultTask)
167. :demo:mergeV3ReleaseAssets                        (com.android.build.gradle.tasks.MergeSourceSetFolders)
168. :demo:compressV3ReleaseAssets                     (com.android.build.gradle.internal.tasks.CompressAssetsTask)
169. :demo:optimizeV3ReleaseResources                  (com.android.build.gradle.internal.tasks.OptimizeResourcesTask)
170. :demo:collectV3ReleaseDependencies                (com.android.build.gradle.internal.tasks.PerModuleReportDependenciesTask)
171. :demo:sdkV3ReleaseDependencyData                  (com.android.build.gradle.internal.tasks.SdkDependencyDataGeneratorTask)
172. :demo:writeV3ReleaseAppMetadata                   (com.android.build.gradle.internal.tasks.AppMetadataTask)
173. :demo:writeV3ReleaseSigningConfigVersions         (com.android.build.gradle.internal.tasks.SigningConfigVersionsWriterTask)
174. :demo:packageV3Release                            (com.android.build.gradle.tasks.PackageApplication)
175. :demo:assembleV3Release                           (org.gradle.api.Task)
176. :demo:assembleRelease                             (org.gradle.api.Task)
177. :demo:assembleV3                                  (org.gradle.api.Task)
178. :demo:assemble                                    (org.gradle.api.DefaultTask)
179. :demo:bundleGmsDebugClasses                       (com.android.build.gradle.internal.feature.BundleAllClasses)
180. :demo:lintAnalyzeGmsDebug                         (com.android.build.gradle.internal.lint.AndroidLintAnalysisTask)
181. :demo:lintGmsDebug                                (com.android.build.gradle.internal.lint.AndroidLintTask)
182. :demo:lint                                        (com.android.build.gradle.internal.lint.AndroidLintGlobalTask)
183. :demo:compileGmsDebugUnitTestKotlin               (org.jetbrains.kotlin.gradle.tasks.KotlinCompileWithWorkers)
184. :demo:preGmsDebugUnitTestBuild                    (com.android.build.gradle.internal.tasks.AndroidVariantTask)
185. :demo:javaPreCompileGmsDebugUnitTest              (com.android.build.gradle.tasks.JavaPreCompileTask)
186. :demo:compileGmsDebugUnitTestJavaWithJavac        (org.gradle.api.tasks.compile.JavaCompile)
187. :demo:processGmsDebugUnitTestJavaRes              (com.android.build.gradle.internal.tasks.ProcessJavaResTask)
188. :demo:testGmsDebugUnitTest                        (com.android.build.gradle.tasks.factory.AndroidUnitTest)
189. :demo:compileGmsReleaseUnitTestKotlin             (org.jetbrains.kotlin.gradle.tasks.KotlinCompileWithWorkers)
190. :demo:preGmsReleaseUnitTestBuild                  (com.android.build.gradle.internal.tasks.AndroidVariantTask)
191. :demo:javaPreCompileGmsReleaseUnitTest            (com.android.build.gradle.tasks.JavaPreCompileTask)
192. :demo:compileGmsReleaseUnitTestJavaWithJavac      (org.gradle.api.tasks.compile.JavaCompile)
193. :demo:processGmsReleaseUnitTestJavaRes            (com.android.build.gradle.internal.tasks.ProcessJavaResTask)
194. :demo:testGmsReleaseUnitTest                      (com.android.build.gradle.tasks.factory.AndroidUnitTest)
195. :demo:bundleV3DebugClasses                        (com.android.build.gradle.internal.feature.BundleAllClasses)
196. :demo:compileV3DebugUnitTestKotlin                (org.jetbrains.kotlin.gradle.tasks.KotlinCompileWithWorkers)
197. :demo:preV3DebugUnitTestBuild                     (com.android.build.gradle.internal.tasks.AndroidVariantTask)
198. :demo:javaPreCompileV3DebugUnitTest               (com.android.build.gradle.tasks.JavaPreCompileTask)
199. :demo:compileV3DebugUnitTestJavaWithJavac         (org.gradle.api.tasks.compile.JavaCompile)
200. :demo:processV3DebugUnitTestJavaRes               (com.android.build.gradle.internal.tasks.ProcessJavaResTask)
201. :demo:testV3DebugUnitTest                         (com.android.build.gradle.tasks.factory.AndroidUnitTest)
202. :demo:compileV3ReleaseUnitTestKotlin              (org.jetbrains.kotlin.gradle.tasks.KotlinCompileWithWorkers)
203. :demo:preV3ReleaseUnitTestBuild                   (com.android.build.gradle.internal.tasks.AndroidVariantTask)
204. :demo:javaPreCompileV3ReleaseUnitTest             (com.android.build.gradle.tasks.JavaPreCompileTask)
205. :demo:compileV3ReleaseUnitTestJavaWithJavac       (org.gradle.api.tasks.compile.JavaCompile)
206. :demo:processV3ReleaseUnitTestJavaRes             (com.android.build.gradle.internal.tasks.ProcessJavaResTask)
207. :demo:testV3ReleaseUnitTest                       (com.android.build.gradle.tasks.factory.AndroidUnitTest)
208. :demo:test                                        (org.gradle.api.Task)
209. :demo:check                                       (org.gradle.api.DefaultTask)
210. :demo:build                                       (org.gradle.api.DefaultTask)
211. :copyGmsToV3Beta                                  (org.gradle.api.tasks.Copy)
212. :library:preBuild                                 (org.gradle.api.DefaultTask)
213. :library:preDebugBuild                            (com.android.build.gradle.internal.tasks.AndroidVariantTask)
214. :library:compileDebugAidl                         (com.android.build.gradle.tasks.AidlCompile)
215. :library:mergeDebugJniLibFolders                  (com.android.build.gradle.tasks.MergeSourceSetFolders)
216. :library:mergeDebugNativeLibs                     (com.android.build.gradle.internal.tasks.MergeNativeLibsTask)
217. :library:stripDebugDebugSymbols                   (com.android.build.gradle.internal.tasks.StripDebugSymbolsTask)
218. :library:copyDebugJniLibsProjectAndLocalJars      (com.android.build.gradle.internal.tasks.LibraryJniLibsTask)
219. :library:compileDebugRenderscript                 (com.android.build.gradle.tasks.RenderscriptCompile)
220. :library:generateDebugBuildConfig                 (com.android.build.gradle.tasks.GenerateBuildConfig)
221. :library:generateDebugResValues                   (com.android.build.gradle.tasks.GenerateResValues)
222. :library:generateDebugResources                   (org.gradle.api.DefaultTask)
223. :library:packageDebugResources                    (com.android.build.gradle.tasks.MergeResources)
224. :library:parseDebugLocalResources                 (com.android.build.gradle.internal.res.ParseLibraryResourcesTask)
225. :library:processDebugManifest                     (com.android.build.gradle.tasks.ProcessLibraryManifest)
226. :library:generateDebugRFile                       (com.android.build.gradle.internal.res.GenerateLibraryRFileTask)
227. :library:compileDebugKotlin                       (org.jetbrains.kotlin.gradle.tasks.KotlinCompileWithWorkers)
228. :library:extractDebugAnnotations                  (com.android.build.gradle.tasks.ExtractAnnotations)
229. :library:javaPreCompileDebug                      (com.android.build.gradle.tasks.JavaPreCompileTask)
230. :library:compileDebugJavaWithJavac                (org.gradle.api.tasks.compile.JavaCompile)
231. :library:mergeDebugGeneratedProguardFiles         (com.android.build.gradle.internal.tasks.MergeFileTask)
232. :library:mergeDebugConsumerProguardFiles          (com.android.build.gradle.internal.tasks.MergeConsumerProguardFilesTask)
233. :library:mergeDebugShaders                        (com.android.build.gradle.tasks.MergeSourceSetFolders)
234. :library:compileDebugShaders                      (com.android.build.gradle.tasks.ShaderCompile)
235. :library:generateDebugAssets                      (org.gradle.api.DefaultTask)
236. :library:packageDebugAssets                       (com.android.build.gradle.tasks.MergeSourceSetFolders)
237. :library:packageDebugRenderscript                 (com.android.build.gradle.internal.tasks.PackageRenderscriptTask)
238. :library:prepareDebugArtProfile                   (com.android.build.gradle.tasks.ProcessLibraryArtProfileTask)
239. :lint-checks:compileKotlin                        (org.jetbrains.kotlin.gradle.tasks.KotlinCompileWithWorkers)
240. :lint-checks:compileJava                          (org.gradle.api.tasks.compile.JavaCompile)
241. :lint-checks:processResources                     (org.gradle.language.jvm.tasks.ProcessResources)
242. :lint-checks:classes                              (org.gradle.api.DefaultTask)
243. :lint-checks:inspectClassesForKotlinIC            (org.jetbrains.kotlin.gradle.tasks.InspectClassesForMultiModuleIC)
244. :lint-checks:jar                                  (org.gradle.api.tasks.bundling.Jar)
245. :library:prepareLintJarForPublish                 (com.android.build.gradle.internal.tasks.PrepareLintJarForPublish)
246. :library:processDebugJavaRes                      (com.android.build.gradle.internal.tasks.ProcessJavaResTask)
247. :library:mergeDebugJavaResource                   (com.android.build.gradle.internal.tasks.MergeJavaResourceTask)
248. :library:syncDebugLibJars                         (com.android.build.gradle.internal.tasks.LibraryAarJarsTask)
249. :library:writeDebugAarMetadata                    (com.android.build.gradle.internal.tasks.AarMetadataTask)
250. :library:bundleDebugAar                           (com.android.build.gradle.tasks.BundleAar)
251. :library:mergeDebugResources                      (com.android.build.gradle.tasks.MergeResources)
252. :library:compileDebugSources                      (org.gradle.api.Task)
253. :library:assembleDebug                            (org.gradle.api.Task)
254. :library:preReleaseBuild                          (com.android.build.gradle.internal.tasks.AndroidVariantTask)
255. :library:compileReleaseAidl                       (com.android.build.gradle.tasks.AidlCompile)
256. :library:mergeReleaseJniLibFolders                (com.android.build.gradle.tasks.MergeSourceSetFolders)
257. :library:mergeReleaseNativeLibs                   (com.android.build.gradle.internal.tasks.MergeNativeLibsTask)
258. :library:stripReleaseDebugSymbols                 (com.android.build.gradle.internal.tasks.StripDebugSymbolsTask)
259. :library:copyReleaseJniLibsProjectAndLocalJars    (com.android.build.gradle.internal.tasks.LibraryJniLibsTask)
260. :library:compileReleaseRenderscript               (com.android.build.gradle.tasks.RenderscriptCompile)
261. :library:generateReleaseBuildConfig               (com.android.build.gradle.tasks.GenerateBuildConfig)
262. :library:generateReleaseResValues                 (com.android.build.gradle.tasks.GenerateResValues)
263. :library:generateReleaseResources                 (org.gradle.api.DefaultTask)
264. :library:packageReleaseResources                  (com.android.build.gradle.tasks.MergeResources)
265. :library:parseReleaseLocalResources               (com.android.build.gradle.internal.res.ParseLibraryResourcesTask)
266. :library:processReleaseManifest                   (com.android.build.gradle.tasks.ProcessLibraryManifest)
267. :library:generateReleaseRFile                     (com.android.build.gradle.internal.res.GenerateLibraryRFileTask)
268. :library:compileReleaseKotlin                     (org.jetbrains.kotlin.gradle.tasks.KotlinCompileWithWorkers)
269. :library:extractReleaseAnnotations                (com.android.build.gradle.tasks.ExtractAnnotations)
270. :library:javaPreCompileRelease                    (com.android.build.gradle.tasks.JavaPreCompileTask)
271. :library:compileReleaseJavaWithJavac              (org.gradle.api.tasks.compile.JavaCompile)
272. :library:mergeReleaseGeneratedProguardFiles       (com.android.build.gradle.internal.tasks.MergeFileTask)
273. :library:mergeReleaseConsumerProguardFiles        (com.android.build.gradle.internal.tasks.MergeConsumerProguardFilesTask)
274. :library:mergeReleaseShaders                      (com.android.build.gradle.tasks.MergeSourceSetFolders)
275. :library:compileReleaseShaders                    (com.android.build.gradle.tasks.ShaderCompile)
276. :library:generateReleaseAssets                    (org.gradle.api.DefaultTask)
277. :library:packageReleaseAssets                     (com.android.build.gradle.tasks.MergeSourceSetFolders)
278. :library:packageReleaseRenderscript               (com.android.build.gradle.internal.tasks.PackageRenderscriptTask)
279. :library:prepareReleaseArtProfile                 (com.android.build.gradle.tasks.ProcessLibraryArtProfileTask)
280. :library:processReleaseJavaRes                    (com.android.build.gradle.internal.tasks.ProcessJavaResTask)
281. :library:mergeReleaseJavaResource                 (com.android.build.gradle.internal.tasks.MergeJavaResourceTask)
282. :library:syncReleaseLibJars                       (com.android.build.gradle.internal.tasks.LibraryAarJarsTask)
283. :library:writeReleaseAarMetadata                  (com.android.build.gradle.internal.tasks.AarMetadataTask)
284. :library:bundleReleaseAar                         (com.android.build.gradle.tasks.BundleAar)
285. :library:mergeReleaseResources                    (com.android.build.gradle.tasks.MergeResources)
286. :library:compileReleaseSources                    (org.gradle.api.Task)
287. :library:verifyReleaseResources                   (com.android.build.gradle.tasks.VerifyLibraryResourcesTask)
288. :library:assembleRelease                          (org.gradle.api.Task)
289. :library:assemble                                 (org.gradle.api.DefaultTask)
290. workNodeAction6aec12b0                            (org.gradle.execution.plan.ActionNode)
291. :library:writeDebugLintModelMetadata              (com.android.build.gradle.internal.tasks.LintModelMetadataTask)
292. :library:bundleDebugLocalLintAar                  (com.android.build.gradle.tasks.BundleAar)
293. :library:transformationStep14b85ee2               (org.gradle.api.internal.artifacts.transform.TransformationNode$InitialTransformationNode)
294. :library:transformationStep27091bff               (org.gradle.api.internal.artifacts.transform.TransformationNode$InitialTransformationNode)
295. :library:bundleLibResDebug                        (com.android.build.gradle.internal.tasks.BundleLibraryJavaRes)
296. :library:bundleLibRuntimeToJarDebug               (com.android.build.gradle.internal.tasks.BundleLibraryClassesJar)
297. :library:createFullJarDebug                       (com.android.build.gradle.tasks.ZipMergingTask)
298. :library:extractProguardFiles                     (com.android.build.gradle.internal.tasks.ExtractProguardFiles)
299. :library:lintAnalyzeDebug                         (com.android.build.gradle.internal.lint.AndroidLintAnalysisTask)
300. :library:lintDebug                                (com.android.build.gradle.internal.lint.AndroidLintTask)
301. :library:lint                                     (com.android.build.gradle.internal.lint.AndroidLintGlobalTask)
302. :library:bundleLibCompileToJarDebug               (com.android.build.gradle.internal.tasks.BundleLibraryClassesJar)
303. :library:preDebugUnitTestBuild                    (com.android.build.gradle.internal.tasks.AndroidVariantTask)
304. :library:checkDebugUnitTestAarMetadata            (com.android.build.gradle.internal.tasks.CheckAarMetadataTask)
305. :library:compileDebugLibraryResources             (com.android.build.gradle.tasks.CompileLibraryResourcesTask)
306. :library:generateDebugUnitTestResValues           (com.android.build.gradle.tasks.GenerateResValues)
307. :library:generateDebugUnitTestResources           (org.gradle.api.DefaultTask)
308. :library:mergeDebugUnitTestResources              (com.android.build.gradle.tasks.MergeResources)
309. :library:extractDeepLinksDebug                    (com.android.build.gradle.tasks.ExtractDeepLinksTask)
310. :library:processDebugUnitTestManifest             (com.android.build.gradle.tasks.ProcessTestManifest)
311. :library:processDebugUnitTestResources            (com.android.build.gradle.internal.res.LinkApplicationAndroidResourcesTask)
312. :library:compileDebugUnitTestKotlin               (org.jetbrains.kotlin.gradle.tasks.KotlinCompileWithWorkers)
313. :library:javaPreCompileDebugUnitTest              (com.android.build.gradle.tasks.JavaPreCompileTask)
314. :library:compileDebugUnitTestJavaWithJavac        (org.gradle.api.tasks.compile.JavaCompile)
315. :library:mergeDebugAssets                         (com.android.build.gradle.tasks.MergeSourceSetFolders)
316. :library:generateDebugUnitTestAssets              (org.gradle.api.DefaultTask)
317. :library:mergeDebugUnitTestAssets                 (com.android.build.gradle.tasks.MergeSourceSetFolders)
318. :library:packageDebugUnitTestForUnitTest          (com.android.build.gradle.internal.tasks.PackageForUnitTest)
319. :library:generateDebugUnitTestConfig              (com.android.build.gradle.tasks.GenerateTestConfig)
320. :library:processDebugUnitTestJavaRes              (com.android.build.gradle.internal.tasks.ProcessJavaResTask)
321. :library:testDebugUnitTest                        (com.android.build.gradle.tasks.factory.AndroidUnitTest)
322. :library:bundleLibResRelease                      (com.android.build.gradle.internal.tasks.BundleLibraryJavaRes)
323. :library:bundleLibRuntimeToJarRelease             (com.android.build.gradle.internal.tasks.BundleLibraryClassesJar)
324. :library:bundleLibCompileToJarRelease             (com.android.build.gradle.internal.tasks.BundleLibraryClassesJar)
325. :library:preReleaseUnitTestBuild                  (com.android.build.gradle.internal.tasks.AndroidVariantTask)
326. :library:checkReleaseUnitTestAarMetadata          (com.android.build.gradle.internal.tasks.CheckAarMetadataTask)
327. :library:compileReleaseLibraryResources           (com.android.build.gradle.tasks.CompileLibraryResourcesTask)
328. :library:generateReleaseUnitTestResValues         (com.android.build.gradle.tasks.GenerateResValues)
329. :library:generateReleaseUnitTestResources         (org.gradle.api.DefaultTask)
330. :library:mergeReleaseUnitTestResources            (com.android.build.gradle.tasks.MergeResources)
331. :library:extractDeepLinksRelease                  (com.android.build.gradle.tasks.ExtractDeepLinksTask)
332. :library:processReleaseUnitTestManifest           (com.android.build.gradle.tasks.ProcessTestManifest)
333. :library:processReleaseUnitTestResources          (com.android.build.gradle.internal.res.LinkApplicationAndroidResourcesTask)
334. :library:compileReleaseUnitTestKotlin             (org.jetbrains.kotlin.gradle.tasks.KotlinCompileWithWorkers)
335. :library:javaPreCompileReleaseUnitTest            (com.android.build.gradle.tasks.JavaPreCompileTask)
336. :library:compileReleaseUnitTestJavaWithJavac      (org.gradle.api.tasks.compile.JavaCompile)
337. :library:mergeReleaseAssets                       (com.android.build.gradle.tasks.MergeSourceSetFolders)
338. :library:generateReleaseUnitTestAssets            (org.gradle.api.DefaultTask)
339. :library:mergeReleaseUnitTestAssets               (com.android.build.gradle.tasks.MergeSourceSetFolders)
340. :library:packageReleaseUnitTestForUnitTest        (com.android.build.gradle.internal.tasks.PackageForUnitTest)
341. :library:generateReleaseUnitTestConfig            (com.android.build.gradle.tasks.GenerateTestConfig)
342. :library:processReleaseUnitTestJavaRes            (com.android.build.gradle.internal.tasks.ProcessJavaResTask)
343. :library:testReleaseUnitTest                      (com.android.build.gradle.tasks.factory.AndroidUnitTest)
344. :library:test                                     (org.gradle.api.Task)
345. :library:check                                    (org.gradle.api.DefaultTask)
346. :library:build                                    (org.gradle.api.DefaultTask)
347. :library-v3:preBuild                              (org.gradle.api.DefaultTask)
348. :library-v3:preDebugBuild                         (com.android.build.gradle.internal.tasks.AndroidVariantTask)
349. :library-v3:compileDebugAidl                      (com.android.build.gradle.tasks.AidlCompile)
350. :library-v3:mergeDebugJniLibFolders               (com.android.build.gradle.tasks.MergeSourceSetFolders)
351. :library-v3:mergeDebugNativeLibs                  (com.android.build.gradle.internal.tasks.MergeNativeLibsTask)
352. :library-v3:stripDebugDebugSymbols                (com.android.build.gradle.internal.tasks.StripDebugSymbolsTask)
353. :library-v3:copyDebugJniLibsProjectAndLocalJars   (com.android.build.gradle.internal.tasks.LibraryJniLibsTask)
354. :library-v3:compileDebugRenderscript              (com.android.build.gradle.tasks.RenderscriptCompile)
355. :library-v3:generateDebugBuildConfig              (com.android.build.gradle.tasks.GenerateBuildConfig)
356. :library-v3:generateDebugResValues                (com.android.build.gradle.tasks.GenerateResValues)
357. :library-v3:generateDebugResources                (org.gradle.api.DefaultTask)
358. :library-v3:packageDebugResources                 (com.android.build.gradle.tasks.MergeResources)
359. :library-v3:parseDebugLocalResources              (com.android.build.gradle.internal.res.ParseLibraryResourcesTask)
360. :library-v3:processDebugManifest                  (com.android.build.gradle.tasks.ProcessLibraryManifest)
361. :library-v3:generateDebugRFile                    (com.android.build.gradle.internal.res.GenerateLibraryRFileTask)
362. :library-v3:compileDebugKotlin                    (org.jetbrains.kotlin.gradle.tasks.KotlinCompileWithWorkers)
363. :library-v3:extractDebugAnnotations               (com.android.build.gradle.tasks.ExtractAnnotations)
364. :library-v3:javaPreCompileDebug                   (com.android.build.gradle.tasks.JavaPreCompileTask)
365. :library-v3:compileDebugJavaWithJavac             (org.gradle.api.tasks.compile.JavaCompile)
366. :library-v3:mergeDebugGeneratedProguardFiles      (com.android.build.gradle.internal.tasks.MergeFileTask)
367. :library-v3:mergeDebugConsumerProguardFiles       (com.android.build.gradle.internal.tasks.MergeConsumerProguardFilesTask)
368. :library-v3:mergeDebugShaders                     (com.android.build.gradle.tasks.MergeSourceSetFolders)
369. :library-v3:compileDebugShaders                   (com.android.build.gradle.tasks.ShaderCompile)
370. :library-v3:generateDebugAssets                   (org.gradle.api.DefaultTask)
371. :library-v3:packageDebugAssets                    (com.android.build.gradle.tasks.MergeSourceSetFolders)
372. :library-v3:packageDebugRenderscript              (com.android.build.gradle.internal.tasks.PackageRenderscriptTask)
373. :library-v3:prepareDebugArtProfile                (com.android.build.gradle.tasks.ProcessLibraryArtProfileTask)
374. :library-v3:prepareLintJarForPublish              (com.android.build.gradle.internal.tasks.PrepareLintJarForPublish)
375. :library-v3:processDebugJavaRes                   (com.android.build.gradle.internal.tasks.ProcessJavaResTask)
376. :library-v3:mergeDebugJavaResource                (com.android.build.gradle.internal.tasks.MergeJavaResourceTask)
377. :library-v3:syncDebugLibJars                      (com.android.build.gradle.internal.tasks.LibraryAarJarsTask)
378. :library-v3:writeDebugAarMetadata                 (com.android.build.gradle.internal.tasks.AarMetadataTask)
379. :library-v3:bundleDebugAar                        (com.android.build.gradle.tasks.BundleAar)
380. :library-v3:mergeDebugResources                   (com.android.build.gradle.tasks.MergeResources)
381. :library-v3:compileDebugSources                   (org.gradle.api.Task)
382. :library-v3:assembleDebug                         (org.gradle.api.Task)
383. :library-v3:preReleaseBuild                       (com.android.build.gradle.internal.tasks.AndroidVariantTask)
384. :library-v3:compileReleaseAidl                    (com.android.build.gradle.tasks.AidlCompile)
385. :library-v3:mergeReleaseJniLibFolders             (com.android.build.gradle.tasks.MergeSourceSetFolders)
386. :library-v3:mergeReleaseNativeLibs                (com.android.build.gradle.internal.tasks.MergeNativeLibsTask)
387. :library-v3:stripReleaseDebugSymbols              (com.android.build.gradle.internal.tasks.StripDebugSymbolsTask)
388. :library-v3:copyReleaseJniLibsProjectAndLocalJars (com.android.build.gradle.internal.tasks.LibraryJniLibsTask)
389. :library-v3:compileReleaseRenderscript            (com.android.build.gradle.tasks.RenderscriptCompile)
390. :library-v3:generateReleaseBuildConfig            (com.android.build.gradle.tasks.GenerateBuildConfig)
391. :library-v3:generateReleaseResValues              (com.android.build.gradle.tasks.GenerateResValues)
392. :library-v3:generateReleaseResources              (org.gradle.api.DefaultTask)
393. :library-v3:packageReleaseResources               (com.android.build.gradle.tasks.MergeResources)
394. :library-v3:parseReleaseLocalResources            (com.android.build.gradle.internal.res.ParseLibraryResourcesTask)
395. :library-v3:processReleaseManifest                (com.android.build.gradle.tasks.ProcessLibraryManifest)
396. :library-v3:generateReleaseRFile                  (com.android.build.gradle.internal.res.GenerateLibraryRFileTask)
397. :library-v3:compileReleaseKotlin                  (org.jetbrains.kotlin.gradle.tasks.KotlinCompileWithWorkers)
398. :library-v3:extractReleaseAnnotations             (com.android.build.gradle.tasks.ExtractAnnotations)
399. :library-v3:javaPreCompileRelease                 (com.android.build.gradle.tasks.JavaPreCompileTask)
400. :library-v3:compileReleaseJavaWithJavac           (org.gradle.api.tasks.compile.JavaCompile)
401. :library-v3:mergeReleaseGeneratedProguardFiles    (com.android.build.gradle.internal.tasks.MergeFileTask)
402. :library-v3:mergeReleaseConsumerProguardFiles     (com.android.build.gradle.internal.tasks.MergeConsumerProguardFilesTask)
403. :library-v3:mergeReleaseShaders                   (com.android.build.gradle.tasks.MergeSourceSetFolders)
404. :library-v3:compileReleaseShaders                 (com.android.build.gradle.tasks.ShaderCompile)
405. :library-v3:generateReleaseAssets                 (org.gradle.api.DefaultTask)
406. :library-v3:packageReleaseAssets                  (com.android.build.gradle.tasks.MergeSourceSetFolders)
407. :library-v3:packageReleaseRenderscript            (com.android.build.gradle.internal.tasks.PackageRenderscriptTask)
408. :library-v3:prepareReleaseArtProfile              (com.android.build.gradle.tasks.ProcessLibraryArtProfileTask)
409. :library-v3:processReleaseJavaRes                 (com.android.build.gradle.internal.tasks.ProcessJavaResTask)
410. :library-v3:mergeReleaseJavaResource              (com.android.build.gradle.internal.tasks.MergeJavaResourceTask)
411. :library-v3:syncReleaseLibJars                    (com.android.build.gradle.internal.tasks.LibraryAarJarsTask)
412. :library-v3:writeReleaseAarMetadata               (com.android.build.gradle.internal.tasks.AarMetadataTask)
413. :library-v3:bundleReleaseAar                      (com.android.build.gradle.tasks.BundleAar)
414. :library-v3:mergeReleaseResources                 (com.android.build.gradle.tasks.MergeResources)
415. :library-v3:compileReleaseSources                 (org.gradle.api.Task)
416. :library-v3:verifyReleaseResources                (com.android.build.gradle.tasks.VerifyLibraryResourcesTask)
417. :library-v3:assembleRelease                       (org.gradle.api.Task)
418. :library-v3:assemble                              (org.gradle.api.DefaultTask)
419. :library-v3:workNodeAction66d2fa30                (org.gradle.execution.plan.ActionNode)
420. :library-v3:writeDebugLintModelMetadata           (com.android.build.gradle.internal.tasks.LintModelMetadataTask)
421. :library-v3:bundleDebugLocalLintAar               (com.android.build.gradle.tasks.BundleAar)
422. :library-v3:transformationStep1e035feb            (org.gradle.api.internal.artifacts.transform.TransformationNode$InitialTransformationNode)
423. :library-v3:transformationStep911ff86             (org.gradle.api.internal.artifacts.transform.TransformationNode$InitialTransformationNode)
424. :library-v3:bundleLibResDebug                     (com.android.build.gradle.internal.tasks.BundleLibraryJavaRes)
425. :library-v3:bundleLibRuntimeToJarDebug            (com.android.build.gradle.internal.tasks.BundleLibraryClassesJar)
426. :library-v3:createFullJarDebug                    (com.android.build.gradle.tasks.ZipMergingTask)
427. :library-v3:extractProguardFiles                  (com.android.build.gradle.internal.tasks.ExtractProguardFiles)
428. :library-v3:lintAnalyzeDebug                      (com.android.build.gradle.internal.lint.AndroidLintAnalysisTask)
429. :library-v3:lintDebug                             (com.android.build.gradle.internal.lint.AndroidLintTask)
430. :library-v3:lint                                  (com.android.build.gradle.internal.lint.AndroidLintGlobalTask)
431. :library-v3:bundleLibCompileToJarDebug            (com.android.build.gradle.internal.tasks.BundleLibraryClassesJar)
432. :library-v3:preDebugUnitTestBuild                 (com.android.build.gradle.internal.tasks.AndroidVariantTask)
433. :library-v3:checkDebugUnitTestAarMetadata         (com.android.build.gradle.internal.tasks.CheckAarMetadataTask)
434. :library-v3:compileDebugLibraryResources          (com.android.build.gradle.tasks.CompileLibraryResourcesTask)
435. :library-v3:generateDebugUnitTestResValues        (com.android.build.gradle.tasks.GenerateResValues)
436. :library-v3:generateDebugUnitTestResources        (org.gradle.api.DefaultTask)
437. :library-v3:mergeDebugUnitTestResources           (com.android.build.gradle.tasks.MergeResources)
438. :library-v3:extractDeepLinksDebug                 (com.android.build.gradle.tasks.ExtractDeepLinksTask)
439. :library-v3:processDebugUnitTestManifest          (com.android.build.gradle.tasks.ProcessTestManifest)
440. :library-v3:processDebugUnitTestResources         (com.android.build.gradle.internal.res.LinkApplicationAndroidResourcesTask)
441. :library-v3:compileDebugUnitTestKotlin            (org.jetbrains.kotlin.gradle.tasks.KotlinCompileWithWorkers)
442. :library-v3:javaPreCompileDebugUnitTest           (com.android.build.gradle.tasks.JavaPreCompileTask)
443. :library-v3:compileDebugUnitTestJavaWithJavac     (org.gradle.api.tasks.compile.JavaCompile)
444. :library-v3:mergeDebugAssets                      (com.android.build.gradle.tasks.MergeSourceSetFolders)
445. :library-v3:generateDebugUnitTestAssets           (org.gradle.api.DefaultTask)
446. :library-v3:mergeDebugUnitTestAssets              (com.android.build.gradle.tasks.MergeSourceSetFolders)
447. :library-v3:packageDebugUnitTestForUnitTest       (com.android.build.gradle.internal.tasks.PackageForUnitTest)
448. :library-v3:generateDebugUnitTestConfig           (com.android.build.gradle.tasks.GenerateTestConfig)
449. :library-v3:processDebugUnitTestJavaRes           (com.android.build.gradle.internal.tasks.ProcessJavaResTask)
450. :library-v3:testDebugUnitTest                     (com.android.build.gradle.tasks.factory.AndroidUnitTest)
451. :library-v3:bundleLibResRelease                   (com.android.build.gradle.internal.tasks.BundleLibraryJavaRes)
452. :library-v3:bundleLibRuntimeToJarRelease          (com.android.build.gradle.internal.tasks.BundleLibraryClassesJar)
453. :library-v3:bundleLibCompileToJarRelease          (com.android.build.gradle.internal.tasks.BundleLibraryClassesJar)
454. :library-v3:preReleaseUnitTestBuild               (com.android.build.gradle.internal.tasks.AndroidVariantTask)
455. :library-v3:checkReleaseUnitTestAarMetadata       (com.android.build.gradle.internal.tasks.CheckAarMetadataTask)
456. :library-v3:compileReleaseLibraryResources        (com.android.build.gradle.tasks.CompileLibraryResourcesTask)
457. :library-v3:generateReleaseUnitTestResValues      (com.android.build.gradle.tasks.GenerateResValues)
458. :library-v3:generateReleaseUnitTestResources      (org.gradle.api.DefaultTask)
459. :library-v3:mergeReleaseUnitTestResources         (com.android.build.gradle.tasks.MergeResources)
460. :library-v3:extractDeepLinksRelease               (com.android.build.gradle.tasks.ExtractDeepLinksTask)
461. :library-v3:processReleaseUnitTestManifest        (com.android.build.gradle.tasks.ProcessTestManifest)
462. :library-v3:processReleaseUnitTestResources       (com.android.build.gradle.internal.res.LinkApplicationAndroidResourcesTask)
463. :library-v3:compileReleaseUnitTestKotlin          (org.jetbrains.kotlin.gradle.tasks.KotlinCompileWithWorkers)
464. :library-v3:javaPreCompileReleaseUnitTest         (com.android.build.gradle.tasks.JavaPreCompileTask)
465. :library-v3:compileReleaseUnitTestJavaWithJavac   (org.gradle.api.tasks.compile.JavaCompile)
466. :library-v3:mergeReleaseAssets                    (com.android.build.gradle.tasks.MergeSourceSetFolders)
467. :library-v3:generateReleaseUnitTestAssets         (org.gradle.api.DefaultTask)
468. :library-v3:mergeReleaseUnitTestAssets            (com.android.build.gradle.tasks.MergeSourceSetFolders)
469. :library-v3:packageReleaseUnitTestForUnitTest     (com.android.build.gradle.internal.tasks.PackageForUnitTest)
470. :library-v3:generateReleaseUnitTestConfig         (com.android.build.gradle.tasks.GenerateTestConfig)
471. :library-v3:processReleaseUnitTestJavaRes         (com.android.build.gradle.internal.tasks.ProcessJavaResTask)
472. :library-v3:testReleaseUnitTest                   (com.android.build.gradle.tasks.factory.AndroidUnitTest)
473. :library-v3:test                                  (org.gradle.api.Task)
474. :library-v3:check                                 (org.gradle.api.DefaultTask)
475. :library-v3:build                                 (org.gradle.api.DefaultTask)
476. :lint-checks:assemble                             (org.gradle.api.DefaultTask)
477. :lint-checks:lintAnalyze                          (com.android.build.gradle.internal.lint.AndroidLintAnalysisTask)
478. :lint-checks:lint                                 (com.android.build.gradle.internal.lint.AndroidLintTask)
479. :lint-checks:copyAndroidLintReports               (com.android.build.gradle.internal.lint.AndroidLintCopyReportTask)
480. :lint-checks:compileTestKotlin                    (org.jetbrains.kotlin.gradle.tasks.KotlinCompileWithWorkers)
481. :lint-checks:compileTestJava                      (org.gradle.api.tasks.compile.JavaCompile)
482. :lint-checks:processTestResources                 (org.gradle.language.jvm.tasks.ProcessResources)
483. :lint-checks:testClasses                          (org.gradle.api.DefaultTask)
484. :lint-checks:test                                 (org.gradle.api.tasks.testing.Test)
485. :lint-checks:check                                (org.gradle.api.DefaultTask)
486. :lint-checks:build                                (org.gradle.api.DefaultTask)

Items in gray represent nodes which are not backed by tasks but by transformation or action nodes added by Gradle.

BUILD SUCCESSFUL in 2s
1 actionable task: 1 executed
```

Reply:
Gotcha, thanks for clarifying!